### DISCUSSION TITLE
Alpine.js needs a website and a logo

### DISCUSSION THREAD

**Discussion by ghost** [alpinejs/alpine#1360] [3. Feature ideas]
Please get an official website and a logo for Alpine.js

**Comment by KevinBatdorf** [comment]
I believe Caleb has one coming soon.

For sake of discussion,
What do you think should be included in the new website? 
And what sort of logo design do you like?

**Reply by ghost** [reply]
I would like to see a documentation section like [this](https://tailwindcss.com/docs) and a logo like [this](https://dribbble.com/shots/15765466-Mountain) 🏔️
